assets/queries/terraform/alicloud/oss_bucket_actiontrail_destination_public/metadata.json

```
@@ -0,0 +1,11 @@
+{
+  "id": "69b5d7da-a5db-4db9-a42e-90b65d0efb0b",
+  "queryName": "OSS Bucket Of Actiontrail Trail Destination Is Public",
```

```suggestion
  "queryName": "ActionTrail Trail OSS Bucket is Publicly Accessible",
```

Please, also update in the query folder name and in the `assets/queries/terraform/alicloud/oss_bucket_actiontrail_destination_public/test/positive_expected_result.json`